cmd/crictl/config.go

```diff
@@ -28,20 +28,20 @@ import (
 	"github.com/kubernetes-sigs/cri-tools/pkg/common"
 )
 
-const configDesc = `Get and set crictl options.
-
-The options are as follows:
+var configCommand = &cli.Command{
+	Name:  "config",
+	Usage: "Get and set crictl client configuration options",
+	ArgsUsage: `[<crictl options>]
 
-- runtime-endpoint: Container runtime endpoint
-- image-endpoint: Image endpoint
-- timeout: Timeout of connecting to server
-- debug: Enable debug output
-`
+EXAMPLE:
+   crictl config --set debug=true
 
-var configCommand = &cli.Command{
-	Name:                   "config",
-	Usage:                  configDesc,
-	ArgsUsage:              "[<options>]",
+CRICTL OPTIONS:
+	 runtime-endpoint:	Container runtime endpoint
+	 image-endpoint:	Image endpoint
+	 timeout:	Timeout of connecting to server
+	 debug:	Enable debug output
+	 pull-image-on-create: 	Enable pulling image on create requests`,
```

Comment:
nit: removed extra whitespace, left the tab

```suggestion
	 pull-image-on-create:	Enable pulling image on create requests`,
```

Reply:
I'll check that out..